sdk/communication/azure-communication-rooms/tests/test_rooms_client.py

```diff
@@ -82,6 +80,7 @@ def test_create_room_only_participants(self):
         self.verify_successful_room_response(response=response)
 
     @pytest.mark.live_test_only
+    @recorded_by_proxy
```

Comment:
For tests that are only run in live mode, we can usually remove the `@recorded_by_proxy` decorator since we don't need the recordings for playback testing. If you prefer to keep the recordings anyway for reference purposes though, keeping the decorator/recording is fine

Reply:
Great point. Let's keep the recordings anyway just so we can verify that the requests/responses are what we'd expect.